src/components/SystemMessageList/SystemMessageList.tsx

```diff
@@ -0,0 +1,60 @@
+import React, {useEffect, useState} from 'react';
+import {SystemMessageNotification, SystemMessageNotificationTypes} from '../SystemMessageNotification/SystemMessageNotification';
+import {getSystemMessages} from '../../api';
+import {useMount} from '../../hooks/useMount';
+
+export interface SystemMessageList {
```

Comment:
Create a new file called `SystemMessage.ts` in the `src/types` folder and move this interface into it. Rename the interface to `SystemMessage` - It's not a list.

Reply:
In webstorm you can have it automatically fix all usages by clicking on `SystemMessageList`, press Shift + F6 - it will highlight, backspace and type it out to be `SystemMessage` and press enter. That will fix all the usages to match the new name. Then click the name again and just press F6, this time it will prompt for the file to move it to, and make the tail end of the path match: `src\types\SystemMessage.ts` note: keep the C:\<folder to project> prefix in there. ex. 
![image](https://user-images.githubusercontent.com/4037224/142087045-94fd0a11-99a3-4844-9712-15f3c4ee5e59.png)